src/coprocessor/endpoint.rs

```diff
@@ -264,6 +264,11 @@ impl<E: Engine> Endpoint<E> {
                 self.check_memory_locks(&req_ctx)?;
                 let quota_limiter = self.quota_limiter.clone();
 
+                // Enable quota limiter auto tune for auto analyze when needed
+                if analyze.get_flags() & REQ_FLAG_TIDB_SYSSESSION > 0 {
+                    quota_limiter.set_auto_tune(true);
```

Comment:
`quota_limiter` is shared, so all analyze tasks regardless of their session number will be auto_tuned from now on.